src/redux/categories/categories.js

```diff
@@ -5,6 +5,7 @@ import { STATUS_CATEGORIES } from '../../constants';
 export default function categoriesReducer(state = {}, action) {
   switch (action.type) {
     case STATUS_CATEGORIES:
+      console.log('Under construction');
```

Comment:
- [ ] Please, remove this debugging piece of code `console.log` in order to improve the cleanliness of the repo.

Reply:
@Meltrust Thank you so much for that feedback, I forgot to delete it. I appreciate it!  😄